cmd/login.go

```diff
@@ -29,6 +29,7 @@ import (
 // Login starts the login handshake with GitHub and okteto
 func Login() *cobra.Command {
 	token := ""
+	force := false
```

Comment:
what's the use case for this? if you pass the url, there is no ask for confirmation, right?